backend/runners/load_representative_points.py

```diff
@@ -99,6 +100,8 @@ def _main(kwargs):
 
     _insert_service_areas(json_features)
     _insert_representative_population_points(json_features)
+    maintenance.vacuum()
```

Comment:
Maybe a good time to implement an ENV variable and bypass this in TEST?

Reply:
TODO